src/languages/en.js

```diff
@@ -582,4 +582,5 @@ export default {
     goAlert: "GoAlert",
     backupOutdatedWarning: "Deprecated: Since a lot of features added and this backup feature is a bit unmaintained, it cannot generate or restore a complete backup.",
     backupRecommend: "Please backup the volume or the data folder (./data/) directly instead.",
+    IconUrl: "Icon-URL",
```

Comment:
Should this be hyphenated?
```suggestion
    IconUrl: "Icon URL",
```

Reply:
Well, it probably shouldn't for English, that's my German roots coming through ;-) Thanks!